### ISSUE TITLE
App obtains routing hints from local ChannelUpdate

### ISSUE THREAD

**Issue by akumaigorodski** [ACINQ/eclair-mobile#228]
While a correct way would be to obtain them from remote `ChannelUpdate` as peer's routing params may be different, see https://gitter.im/ACINQ/developers?at=5db694bc10bd4128a175540d

**Comment by pm47** [comment]
Yes that's a bug, as a matter of fact we have just discovered it a week ago while working on trampoline routing. There was actually another related bug that we have fixed with https://github.com/ACINQ/eclair/pull/1172. The latter reduces considerably the impact of the former: if peers have different values, there will be a retry and it will work.
